src/features/documents/components/DocumentsList/DocumentsList.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactComponent as IconFileDownload } from '../../../../assets/icons/file-download-icon.svg';
+import { DocumentsProps } from '../types';
+
+export function DocumentsList({
+  list,
+}: {
+  list: DocumentsProps
+}) {
+  return (
+    <div className="documents-list" data-cy="documents-list">
+      <ul className="documents-list__list" data-cy="documents-list-list">
+        {list.map(({ id, name, downloadLink }) => (
```

Comment:
add multiline here and in places where there is more than one attribute (for ex, className and data-cy)